dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -36,82 +38,84 @@
    * Constants.
    */
   private static final ObjectMapper MAPPER = new ObjectMapper();
+  private static final SoftwareProperties config = new SoftwareProperties();
```

Comment:
This should be an instance field so that it may be injected and thus overriden by a config file. See other services for an example.